smtlib-backends-z3/src/SMTLIB/Backends/Z3.hs

```diff
@@ -40,6 +40,20 @@ newtype Config = Config
   { -- | A list of options to set during the solver's initialization.
     -- Each pair is of the form @(paramId, paramValue)@, e.g.
     -- @(":produce-models", "true")@.
+    --
+    -- Note that Z3 has different kinds of parameters, and not all of
+    -- them can be set here. In particular, there are the so called
+    -- global and module parameters with a value that affects
+    -- all solver instances (in our experiments those started after
+    -- changing the parameter). We have found some of these global
```

Comment:
The parenthetic remark doesn't add relevant information (plus it's a little hard to read).

```suggestion
    -- global and module parameters with a value that affects
    -- all solver instances. We have found some of these global
```